### ISSUE TITLE
Synch Error

### ISSUE THREAD

**Issue by sonnicolo83** [sabeechen/hassio-google-drive-backup#269]
###### Description:
```
If you have anything else that could help explain what happened, click "Markdown" above and write it here.
```

 Addon version: `0.101.3`
 Home Assistant Version: `0.115.6`
 Supervisor Version: `245`
 Supervisor Channel: `stable`
 Hassos Version: `4.13`
 Docker Version: `19.03.11`
 Architecture: `armv7`
 Machine: `raspberrypi3`
 Date: `2020-10-02T14:26:31.100498+00:00`
 Timezone: `Europe/Rome`
 Failure Time: `2020-10-02T14:26:27.685106+00:00`
 Last Good Sync: `2020-10-02T14:26:27.056762+00:00`
 ###### Exception:
 ```
 
  addon/backup/util/asynchttpgetter.py:133 (read)
  /site-packages/aiohttp/streams.py:416 (readexactly)
aiohttp.client_exceptions.ClientPayloadError: Response payload is not completed
Whose handling caused:
  addon/backup/model/coordinator.py:148 (_sync)
  addon/backup/model/model.py:162 (sync)
  addon/backup/drive/drivesource.py:150 (save)
  addon/backup/drive/driverequests.py:252 (create)
  addon/backup/util/asynchttpgetter.py:140 (read)
backup.exceptions.exceptions.SupervisorUnexpectedError

 ```
 Snapshots:
 ```
 {
    "HomeAssistant": {
        "snapshots": 4,
        "retained": 0,
        "deletable": 4,
        "name": "HomeAssistant",
        "latest": "2020-10-02T09:16:22Z",
        "size": "58.1 GB"
    },
    "GoogleDrive": {
        "snapshots": 4,
        "retained": 0,
        "deletable": 4,
        "name": "GoogleDrive",
        "latest": "2020-09-18T06:09:15Z",
        "size": "778.5 MB"
    }
}
 ```
 ###### Config:
 ```
 {
    "Setting.DAYS_BETWEEN_SNAPSHOTS": 2.0
}
 ```
 ###### Addon Logs:
 ```
 10-02 15:50:09 INFO [backup.drive.drivesource] Uploading 'Full Snapshot 2020-10-02 11:16:21' to Google Drive
10-02 15:50:09 DEBUG [backup.drive.driverequests] Attempting to resume a previosuly failed upload where we left off
10-02 15:50:09 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/upload/drive/v3/files/?uploadType=resumable&supportsAllDrives=true&upload_id=ABg5-UyiWxZtEx7bZVPl8Usa1Y1C1PHZwrA54KnzstMX3zB_OnZBbN0tHzSH3FpRCgQmXIuMsKYJm28RQxvqZtM7m1s
10-02 15:50:09 DEBUG [backup.drive.driverequests] Resuming upload at byte 0 of 15606691840
10-02 15:50:09 ERROR [backup.model.coordinator] The supervisor gave an unexpected response
10-02 15:50:09 INFO [backup.model.coordinator] I'll try again in an hour
10-02 16:21:00 INFO [backup.model.coordinator] Syncing Snapshots
10-02 16:21:00 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/snapshots
10-02 16:21:00 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/drive/v3/files/18Q2hp7cbfqa3AIXS0bLpcEK0RqQy8oWa/?fields=id%2Cname%2CappProperties%2Csize%2Ctrashed%2CmimeType%2CmodifiedTime%2Ccapabilities%2Cparents&supportsAllDrives=true
10-02 16:21:01 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/drive/v3/files/?q=%2718Q2hp7cbfqa3AIXS0bLpcEK0RqQy8oWa%27+in+parents&fields=nextPageToken%2Cfiles%28id%2Cname%2CappProperties%2Csize%2Ctrashed%2CmimeType%2CmodifiedTime%2Ccapabilities%2Cparents%29&pageSize=100&supportsAllDrives=true&includeItemsFromAllDrives=true&corpora=allDrives
10-02 16:21:01 INFO [backup.drive.drivesource] Uploading 'Full Snapshot 2020-10-02 11:16:21' to Google Drive
10-02 16:21:01 DEBUG [backup.drive.driverequests] Attempting to resume a previosuly failed upload where we left off
10-02 16:21:01 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/upload/drive/v3/files/?uploadType=resumable&supportsAllDrives=true&upload_id=ABg5-UyiWxZtEx7bZVPl8Usa1Y1C1PHZwrA54KnzstMX3zB_OnZBbN0tHzSH3FpRCgQmXIuMsKYJm28RQxvqZtM7m1s
10-02 16:21:01 DEBUG [backup.drive.driverequests] Resuming upload at byte 0 of 15606691840
10-02 16:21:01 ERROR [backup.model.coordinator] The supervisor gave an unexpected response
10-02 16:21:01 INFO [backup.model.coordinator] I'll try again in an hour
10-02 16:21:13 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/addons/self/info
10-02 16:21:13 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/info
10-02 16:21:13 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/core/info
10-02 16:21:13 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/supervisor/info
10-02 16:26:27 INFO [backup.model.coordinator] Syncing Snapshots
10-02 16:26:27 DEBUG [backup.ha.harequests] Making Hassio request: http://hassio/snapshots
10-02 16:26:27 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/drive/v3/files/18Q2hp7cbfqa3AIXS0bLpcEK0RqQy8oWa/?fields=id%2Cname%2CappProperties%2Csize%2Ctrashed%2CmimeType%2CmodifiedTime%2Ccapabilities%2Cparents&supportsAllDrives=true
10-02 16:26:27 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/drive/v3/files/?q=%2718Q2hp7cbfqa3AIXS0bLpcEK0RqQy8oWa%27+in+parents&fields=nextPageToken%2Cfiles%28id%2Cname%2CappProperties%2Csize%2Ctrashed%2CmimeType%2CmodifiedTime%2Ccapabilities%2Cparents%29&pageSize=100&supportsAllDrives=true&includeItemsFromAllDrives=true&corpora=allDrives
10-02 16:26:27 INFO [backup.drive.drivesource] Uploading 'Full Snapshot 2020-10-02 11:16:21' to Google Drive
10-02 16:26:27 DEBUG [backup.drive.driverequests] Attempting to resume a previosuly failed upload where we left off
10-02 16:26:27 DEBUG [backup.drive.driverequests] Making Google Drive request: https://www.googleapis.com/upload/drive/v3/files/?uploadType=resumable&supportsAllDrives=true&upload_id=ABg5-UyiWxZtEx7bZVPl8Usa1Y1C1PHZwrA54KnzstMX3zB_OnZBbN0tHzSH3FpRCgQmXIuMsKYJm28RQxvqZtM7m1s
10-02 16:26:27 DEBUG [backup.drive.driverequests] Resuming upload at byte 0 of 15606691840
10-02 16:26:27 ERROR [backup.model.coordinator] The supervisor gave an unexpected response
10-02 16:26:27 INFO [backup.model.coordinator] I'll try again in an hour
 ```
 ###### Supervisor Logs:
 ```
 [32m20-10-02 12:50:03 INFO (MainThread) [supervisor.api.security] /snapshots/40512f36/info access from cebe7a76_hassio_google_drive_backup[0m
[32m20-10-02 12:50:04 INFO (MainThread) [supervisor.api.security] /snapshots/40512f36/download access from cebe7a76_hassio_google_drive_backup[0m
[32m20-10-02 12:50:04 INFO (MainThread) [supervisor.api.snapshots] Download snapshot 40512f36[0m
[32m20-10-02 12:50:08 INFO (MainThread) [supervisor.api.security] /snapshots access from cebe7a76_hassio_google_drive_backup[0m
[32m20-10-02 12:50:08 INFO (MainThread) [supervisor.api.security] /snapshots/40512f36/download access from cebe7a76_hassio_google_drive_backup[0m
[32m20-10-02 12:50:08 INFO (MainThread) [supervisor.api.snapshots] Download snapshot 40512f36[0m
[32m20-10-02 13:06:14 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token[0m
[32m20-10-02 13:23:17 INFO (MainThread) [supervisor.updater] Fetch update data from https://version.home-assistant.io/stable.json[0m
[32m20-10-02 13:36:23 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token[0m
[32m20-10-02 13:50:08 INFO (MainThread) [supervisor.api.security] /snapshots access from cebe7a76_hassio_google_drive_backup[0m
[32m20-10-02 13:50:09 INFO (MainThread) [supervisor.api.security] /snapshots/40512f36/download access from cebe7a76_hassio_google_drive_backup[0m
[32m20-10-02 13:50:09 INFO (MainThread) [supervisor.api.snapshots] Download snapshot 40512f36[0m
[32m20-10-02 14:01:46 INFO (MainThread) [supervisor.host.info] Update local host information[0m
[32m20-10-02 14:03:40 INFO (MainThread) [supervisor.host.services] Update service information[0m
[32m20-10-02 14:03:41 INFO (MainThread) [supervisor.host.network] Update local network information[0m
[32m20-10-02 14:03:42 INFO (MainThread) [supervisor.host.sound] Update PulseAudio information[0m
Failed to load cookie file from cookie: No such file or directory
[32m20-10-02 14:06:31 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token[0m
[32m20-10-02 14:21:00 INFO (MainThread) [supervisor.api.security] /snapshots access from cebe7a76_hassio_google_drive_backup[0m
[32m20-10-02 14:21:01 INFO (MainThread) [supervisor.api.security] /snapshots/40512f36/download access from cebe7a76_hassio_google_drive_backup[0m
[32m20-10-02 14:21:01 INFO (MainThread) [supervisor.api.snapshots] Download snapshot 40512f36[0m
[32m20-10-02 14:21:13 INFO (MainThread) [supervisor.api.security] /core/info access from cebe7a76_hassio_google_drive_backup[0m
[32m20-10-02 14:21:13 INFO (MainThread) [supervisor.api.security] /supervisor/info access from cebe7a76_hassio_google_drive_backup[0m
[33m20-10-02 14:22:15 WARNING (MainThread) [supervisor.api.ingress] No valid ingress session None[0m
[31m20-10-02 14:25:03 ERROR (MainThread) [supervisor.api.ingress] Ingress error: Cannot write to closing transport[0m
[32m20-10-02 14:26:27 INFO (MainThread) [supervisor.api.security] /snapshots access from cebe7a76_hassio_google_drive_backup[0m
[32m20-10-02 14:26:27 INFO (MainThread) [supervisor.api.security] /snapshots/40512f36/download access from cebe7a76_hassio_google_drive_backup[0m
[32m20-10-02 14:26:27 INFO (MainThread) [supervisor.api.snapshots] Download snapshot 40512f36[0m
[32m20-10-02 14:26:31 INFO (MainThread) [supervisor.api.security] /supervisor/logs access from cebe7a76_hassio_google_drive_backup[0m

 ```
 ###### Home Assistant Core Logs:
 ```
   File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 422, in schedule_update_ha_state
    assert self.hass is not None
AssertionError
2020-10-02 15:00:59 ERROR (Poll status) [pyShelly] Error update device status: 058E88 SHSW-1, , Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/pyShelly/block.py", line 179, in update_status_information
    dev.raise_updated()
  File "/usr/local/lib/python3.8/site-packages/pyShelly/base.py", line 36, in raise_updated
    callback(self)
  File "/config/custom_components/shelly/device.py", line 55, in _updated
    self.schedule_update_ha_state(True)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 422, in schedule_update_ha_state
    assert self.hass is not None
AssertionError
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/pyShelly/block.py", line 179, in update_status_information
    dev.raise_updated()
  File "/usr/local/lib/python3.8/site-packages/pyShelly/base.py", line 36, in raise_updated
    callback(self)
  File "/config/custom_components/shelly/device.py", line 55, in _updated
    self.schedule_update_ha_state(True)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 422, in schedule_update_ha_state
    assert self.hass is not None
AssertionError
2020-10-02 15:07:02 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from ec2-54-198-14-9.compute-1.amazonaws.com (54.198.14.9) (Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36)
2020-10-02 15:08:05 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from ec2-54-86-208-67.compute-1.amazonaws.com (54.86.208.67) (Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36)
2020-10-02 15:58:09 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from ec2-54-198-85-96.compute-1.amazonaws.com (54.198.85.96) (Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36)
2020-10-02 15:58:24 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from ec2-18-212-118-170.compute-1.amazonaws.com (18.212.118.170) (Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36)
2020-10-02 15:58:33 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from ec2-3-89-138-241.compute-1.amazonaws.com (3.89.138.241) (Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36)
2020-10-02 15:59:18 WARNING (MainThread) [homeassistant.components.http.ban] Login attempt or request with invalid authentication from ec2-23-20-55-112.compute-1.amazonaws.com (23.20.55.112) (Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.109 Safari/537.36)

 ```

**Comment by sabeechen** [comment]
Duplicate of #226
This is a bug in the supervisor, I'm currently blocked on it having the supervisor people look into it.

**Comment by sabeechen** [comment]
I'll be posting any new information about this in #226, though the supervisor people tell me they have out a new release that fixes it within a week.
